src/components/OrganizationSaas/Dashboard/EditProfile.tsx

```diff
@@ -0,0 +1,243 @@
+import {
+  AspectRatio,
+  Box,
+  Flex,
+  FormControl,
+  FormLabel,
+  IconButton,
+  Image,
+  Input,
+  Text,
+  Textarea,
+} from '@chakra-ui/react'
+import { Button } from '@vocdoni/chakra-components'
+import { errorToString, useClient, useOrganization } from '@vocdoni/react-providers'
+import { Account } from '@vocdoni/sdk'
+import { useState } from 'react'
+import { FormProvider, SubmitHandler, useForm } from 'react-hook-form'
+import { Trans, useTranslation } from 'react-i18next'
+import { BiTrash } from 'react-icons/bi'
+import { BsFillTrashFill } from 'react-icons/bs'
+import { MdBrowserUpdated } from 'react-icons/md'
+import InputCustom from '~components/Layout/InputCustom'
+import {
+  CountriesTypesSelector,
+  CustomizationLanguageSelector,
+  CustomizationTimeZoneSelector,
+  MembershipSizeTypesSelector,
+  OrganzationTypesSelector,
+} from '~components/Layout/SaasSelector'
+import useDarkMode from '~src/themes/saas/hooks/useDarkMode'
+import fallback from '/assets/default-avatar.png'
+
+interface EditFormFields {
+  name: string
+  description: string
+  avatar: string
+}
```

Comment:
This fields are the same as on the `src/components/Account/EditProfile.tsx` we should not repeat the code

Reply:
31b0ad5